test/busbranch/conftest.py

```diff
@@ -0,0 +1,382 @@
+    acls5_t = _create_terminal(acls5)
+    network.add(acls5_t)
+    network.add(acls5)
+
+    network.connect_terminals(acls4_terminals[1], acls5_t)
+
+    return network
+
+
+@fixture()
+def negligible_impedance_equipment_basic_network(request) -> NetworkService:
+    """
+    all same psli
+
+                               | a5
+                               +
+                               | a4
+      a0           a1    a2    |      a3
+     ----+ nie1 +----+------ nie2 -----------+
+
+    """
+    # Network
+    network = NetworkService()
+
+    nie_constructor = request.param
+
+    # PerLineSequenceImpedance
+    plsi = _create_per_length_sequence_impedance(1.0)
+    network.add(plsi)
+
+    # AcLineSegment0
+    a0 = AcLineSegment(mrid="a0", length=0.0, per_length_sequence_impedance=plsi)
+    a0_t = _create_terminal(a0)
+    network.add(a0_t)
+    network.add(a0)
+
+    # NegligibleImpedanceEquipment1
+    nie1 = nie_constructor("nie1")
+    nie1_ts = _create_terminals(nie1)
+    network.add(nie1)
+    for t in nie1_ts:
+        network.add(t)
+
+    network.connect_terminals(a0_t, nie1_ts[0])
+
+    # AcLineSegment1
+    a1 = AcLineSegment(mrid="a1", length=1.0, per_length_sequence_impedance=plsi)
+    a1_ts = _create_terminals(a1)
+    for t in a1_ts:
+        network.add(t)
+    network.add(a1)
+
+    network.connect_terminals(nie1_ts[1], a1_ts[0])
+
+    # AcLineSegment2
+    a2 = AcLineSegment(mrid="a2", length=2.0, per_length_sequence_impedance=plsi)
+    a2_ts = _create_terminals(a2)
+    for t in a2_ts:
+        network.add(t)
+    network.add(a2)
+
+    network.connect_terminals(a1_ts[1], a2_ts[0])
+
+    # NegligibleImpedanceEquipment2
+    nie2 = nie_constructor("nie2")
+    nie2_ts = _create_terminals(nie2, [PhaseCode.ABC, PhaseCode.ABC, PhaseCode.ABC])
+    network.add(nie2)
+    for t in nie2_ts:
+        network.add(t)
+
+    network.connect_terminals(a2_ts[1], nie2_ts[0])
+
+    # AcLineSegment3
+    a3 = AcLineSegment(mrid="a3", length=3.0, per_length_sequence_impedance=plsi)
+    a3_ts = _create_terminals(a3)
+    for t in a3_ts:
+        network.add(t)
+    network.add(a3)
+
+    network.connect_terminals(nie2_ts[1], a3_ts[0])
+
+    # AcLineSegment4
+    a4 = AcLineSegment(mrid="a4", length=4.0, per_length_sequence_impedance=plsi)
+    a4_ts = _create_terminals(a4)
+    for t in a4_ts:
+        network.add(t)
+    network.add(a4)
+
+    network.connect_terminals(nie2_ts[2], a4_ts[0])
+
+    # AcLineSegment5
+    a5 = AcLineSegment(mrid="a5", length=5.0, per_length_sequence_impedance=plsi)
+    a5_t = _create_terminal(a5)
+    network.add(a5)
+    network.add(a5_t)
+
+    network.connect_terminals(a4_ts[1], a5_t)
+
+    return network
+
+
+def _create_terminal(ce: ConductingEquipment, phases: PhaseCode = PhaseCode.ABC) -> Terminal:
+    return _create_terminals(ce, [phases])[0]
+
+
+def _create_terminals(ce: ConductingEquipment, phases_per_term: List[PhaseCode] = None) -> List[Terminal]:
```

Comment:
these already exist in network_fixtures.py, which aren't in the right place i know, but i want to avoid having multiple create helpers all over the place. can you use those ones and just modify them to suit your needs if they don't already?